Assets/Scripts/Core/Star.cs

```diff
@@ -116,14 +116,19 @@ public class Star : CelestialSphereItem
     // other properties for a star
     public float size;
 
-    public string XByerFlamsteed;
+    public string XBayerFlamsteed; // this is a sequence of characters that combines Bayer and Flamsteed star id
+    public string BayerDesignation;
+    public string FlamsteedDesignation;
 
-    public Star(int Hip, string Constellation, string ProperName, string XBFlamsteed, float RA, float Dec, float Dist, float Mag, float AbsMag, string Spectrum, float ColorIndex)
+    public Star(int Hip, string ConstellationAbbr, string ConstellationFull, string ProperName, string XBFlamsteed, string FlamsteedDes, string BayerDes, float RA, float Dec, float Dist, float Mag, float AbsMag, string Spectrum, float ColorIndex)
     {
-        this.Hip = Hip;
-        this.Constellation = Constellation;
+        this.Hipparcos = Hip;
+        this.Constellation = ConstellationAbbr;
+        this.ConstellationFullName = ConstellationFull;
         this.ProperName = ProperName;
-        this.XByerFlamsteed = XBFlamsteed;
+        this.XBayerFlamsteed = XBFlamsteed;
+        this.FlamsteedDesignation = FlamsteedDes;
+        this.BayerDesignation = BayerDes;
         this.RA = RA;
         this.radianRA = RA * 15 * Mathf.Deg2Rad; // RA in hours, so multiply RA by 15 deg / hr
```

Comment:
we could take this out of the client side and pre-calculate the radian version of the RA in the database?

Reply:
Are there any floating point issues that we would need to consider?

Reply:
Added both the RA and Dec conversions to radians to the database.